.github/workflows/build-and-push.yml

```diff
@@ -27,12 +27,6 @@ jobs:
             quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
             tag: "c9s"
             suffix: "c9s"
-          - dockerfile_path: "1.20"
```

Comment:
No, because Fedora 36 does not contain nginx-1.20. see https://src.fedoraproject.org/rpms/nginx. Only Nginx.122 is available. And we do not use modules.